test/index.js

```diff
@@ -1,8 +1,53 @@
 const expect = require('chai').expect;
+const StreamGenerator = require('stream-generator');
+const StreamTest = require('streamtest');
 
-describe('my thing', function () {
-  it('should work', function () {
-    expect(true).to.be.true;
-    throw new Error('unimplemented');
+function * simplisticByteGenerator () {
+  // spits out bytes, which correspond to ascii chars:
+  // 'M','a','r','s','h','a','l','l'
+  yield 77;
+  yield 97;
+  yield 114;
+  yield 115;
+  yield 104;
+  yield 97;
+  yield 108;
+  yield 108;
```

Comment:
I guess that is one way to do it... you may be better off doing something like...
```js
function * simplisticByteGenerator () {
  for (var c of 'Marshall') yield c.charCodeAt();
}
```
or better yet
```js
function * simplisticByteGenerator (string) {
  for (var c of string) yield c.charCodeAt();
}
```
so that you can pass in the string you are expecting from the calling test.